src/ngraph_builder.cc

```diff
@@ -3937,6 +3937,45 @@ static Status TranslateTileOp(
   return Status::OK();
 }
 
+// Translate TopKV2 Op using ngraph core op TopK
+static Status TranslateTopKV2Op(
+    const Node* op, const std::vector<const Tensor*>& static_input_map,
+    Builder::OpMap& ng_op_map) {
+  shared_ptr<ngraph::Node> ng_input;
+  ValidateInputCount(op, 2);
+  TF_RETURN_IF_ERROR(GetInputNode(ng_op_map, op, 0, &ng_input));
+
+  size_t k_axis = ng_input->get_shape().size() - 1;
+
+  std::vector<int32> ng_k;
+  size_t k;
+  bool sorted = true;
+
+  TF_RETURN_IF_ERROR(GetStaticInputVector(op, 1, static_input_map, &ng_k));
+  k = ng_k[0];
+
+  GetNodeAttr(op->attrs(), "sorted", &sorted);
+
+  // ngraph topk computes topk min if sorted is false
+  // TODO:ngraph team need to add an attribute to always compute max for topkv2
+  if (sorted == false) {
+    return errors::InvalidArgument("topkv2 should compute max, not min");
```

Comment:
For the error message here, maybe we should just say `TopKV2 doesn't support sorted equals False.`

Reply:
sure, will change this.